ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -38,9 +41,12 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
     isReadonly,
     onEditButtonClick,
     onCancelButtonClick,
+    onSave,
   } = props;
 
   const { isEditMode, setEditMode } = useEditMode();
+  const [isSaveDashboard, setSaveDashboard] = useState<boolean>(false);
```

Comment:
i think calling this `isSavingDashboard` might be more clear. I was confused if this meant is dashboard saved at first